packages/utilities/api/utilities.api.ts

```diff
@@ -127,6 +127,11 @@ export function getFirstFocusable(rootElement: HTMLElement,
   currentElement: HTMLElement,
   includeElementsInFocusZones?: boolean): HTMLElement | null;
 
+// @public
+export function getFirstTabbable(rootElement: HTMLElement,

```

Comment:
This looks like an api-extractor generated file! This is surprising to me because my impression was that API extractor isn't presently integrated.

I asked @dzearing and he was confused too. Did you by any chance hand edit this?

Reply:
I did hand edit the file.  Should it be reverted?

Reply:
It is probably ok but just that the file is itself inconsistent with the rest. What we really should do is to delete this file entirely.. and I can do that as part of a separate PR (or you can do it as part of this PR :))

Reply:
So what happened was that David once integrated apiExtractor but he didn't manage to have it running properly so disabled it (for a long time now) and never deleted the remnants :)

Reply:
+ @kenotron for FYI as he is reintroducing api-extractor.

Reply:
I will remove the file entirely.